tests/Rector/Class_/StaticDataProviderClassMethodRector/Source/AbstractClassWithDataProvider.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\Class_\StaticDataProviderClassMethodRector\Source;
+
+use PHPUnit\Framework\TestCase;
+
+abstract class AbstractClassWithDataProvider extends TestCase
+{
+    /**
+     * @dataProvider provideData
+     */
+    public function test(): void
+    {
+
+    }
+
+    abstract public function provideData(): array;
```

Comment:
Will it also change this?

```diff
- abstract public function provideData(): array;
- abstract public static function provideData(): array;
```
?

Reply:
I guess its part of another test case, right?

Reply:
Let's see :) 

https://github.com/rectorphp/rector-phpunit/pull/156/files